app/templates/base.html

```diff
@@ -51,6 +51,14 @@
                 text-align: left;
                 float: right;
                 overflow-y: auto;
+          }
```

Comment:
Didn't this commit already happen in your last PR?

Reply:
Yeah, I had based this branch off of that one prior to the pull request going through, then rebased onto development